### PR TITLE
add unit test for functions of generating names

### LINKED ISSUE #1021: Add unit test for names.go
**What would you like to be added**:

The `names.go` file contains various methods for generating naming, but there is no unit test covering the case, so we need to add unit test to cover it, to avoid errors in subsequent iterative development.

Anyone who is interested could have a try.

/help

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #1021 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/util/names/names_test.go [hunk 1]
```diff
@@ -100,3 +114,193 @@ func TestGenerateBindingReferenceKey(t *testing.T) {
...
+	}
+	for _, test := range tests {
+		got := GenerateEstimatorServiceName(test.clusterName)
+		if got != test.expected {
+			t.Errorf("Test %s failed: expected %v, but got %v", test.name, test.expected, got)
+		}
+	}
+}
+
+func TestGenerateClusterAPISecretName(t *testing.T) {
```
Comment (XiShanYongYe-Chang) [inline pkg/util/names/names_test.go:288]: Here will conflict with #1029 , because that pr delete the func `GenerateClusterAPISecretName`.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

File: pkg/util/names/names_test.go (hunk 1 above)
Comment (RainbowMango) [reply pkg/util/names/names_test.go:288]: Then, let's move this forward after #1029.

Comment (RainbowMango) [reply pkg/util/names/names_test.go:288]: #1029 get merged.

Comment (gy95) [reply pkg/util/names/names_test.go]: > 

fixed.

**Pull request merged**